flutter/lib/src/sentry_flutter.dart

```diff
@@ -89,12 +89,14 @@ mixin SentryFlutter {
     }
 
     // Will enrich events with device context, native packages and integrations
-    if (options.platformChecker.platform.isIOS ||
-        options.platformChecker.platform.isMacOS) {
+    if (!options.platformChecker.isWeb &&
+        (options.platformChecker.platform.isIOS ||
+            options.platformChecker.platform.isMacOS)) {
       integrations.add(LoadContextsIntegration(channel));
     }
 
-    if (options.platformChecker.platform.isAndroid) {
+    if (!options.platformChecker.isWeb &&
+        options.platformChecker.platform.isAndroid) {
```

Comment:
can we double-check the usage of `.isIOS`, `isMacOS`, and `isAndroid` and see if we've not missed anywhere else the `!isWeb` check? if you've not done it yet :)

Reply:
Yep, I'll double check it.

Reply:
Okay, we're good. The only usages are for adding integrations

Reply:
thanks